src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/PolicyController.cs

```diff
@@ -120,5 +121,34 @@ public async Task<ActionResult> GetRolesWithAccess(string org, string app)
 
             return Ok(PolicyHelper.GetRolesWithAccess(policy));
         }
+
+        /// <summary>
+        /// Gets a list of role codes which might give access to an app
+        /// </summary>
+        /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+        /// <param name="app">Application identifier which is unique within an organisation.</param>
+        [AllowAnonymous]
+        [HttpGet("/authorization/api/v1/resourcepolicies/{org}/{app}")]
```

Comment:
Would the route `authorizaion/api/v1/resource/{org}/{app}/policies` be more RESTful?